pkg/controllers/namespace/namespace_controller.go

```diff
@@ -66,10 +66,10 @@ func NewNamespaceController(ctx context.Context, k8sClientset *kubernetes.Client
 		// Filter out only objects that are written by policy controller.
 		for _, profile := range profileList.Items {
 			if strings.HasPrefix(profile.Name, converter.ProfileNameFormat) {
-				// Update the profile's ObjectMeta so that it simply contains the name and namespace.
+				// Update the profile's ObjectMeta so that it simply contains the name.
 				// There is other metadata that we might receive (like resource version) that we don't want to
 				// compare in the cache.
-				profile.ObjectMeta = metav1.ObjectMeta{Name: profile.Name, Namespace: profile.Namespace}
+				profile.ObjectMeta = metav1.ObjectMeta{Name: profile.Name}
```

Comment:
can we make any assertions about the `Namespace`, or is ignoring the contents sufficient?

Reply:
It's not a namespaced object, so the contents will always be `""`